test_bifrost.sh

```diff
@@ -15,7 +17,7 @@ docker run \
         -post $BIFROST_CONFIG_DIR/post.sh \
         -reads $BIFROST_RUN_DIR/samples \
         -meta $BIFROST_RUN_DIR/run_metadata.tsv \
-        -name bifrost_test \
+        -name "bifrost_test" \
```

Comment:
I changed it from date back, this is for the purpose of re-testing already ran components. The goal isn't to put identical data into the DB twice. I realize it is clunky to clear the db and re-run this.